build_tools/pkgci/external_test_suite/attention_and_matmul_spec.mlir

```diff
@@ -469,6 +1054,38 @@ module attributes { transform.with_named_sequence } {
         // Attention.
         @match_attention_len_512 -> @custom_attention_len_512,
         @match_attention -> @custom_attention
+
+        // Matmul.
+        , @match_mmt_2048x10240x1280 -> @apply_op_config
+        , @match_mmt_2048x1280x5120 -> @apply_op_config
+        , @match_mmt_2048x1280x1280 -> @apply_op_config
+        , @match_mmt_8192x5120x640 -> @apply_op_config
+        , @match_mmt_8192x640x2560 -> @apply_op_config
+        , @match_mmt_8192x640x640 -> @apply_op_config
+
+        // Convolution.
+        , @match_conv_2d_nhwc_hwcf_2x32x32x1280x3x3x640 -> @apply_op_config
+        , @match_conv_2d_nhwc_hwcf_2x32x32x1280x3x3x1280 -> @apply_op_config
+        , @match_conv_2d_nhwc_hwcf_2x32x32x1280x3x3x1920 -> @apply_op_config
+        , @match_conv_2d_nhwc_hwcf_2x32x32x1280x3x3x2560 -> @apply_op_config
+        , @match_conv_2d_nhwc_hwcf_2x64x64x640x3x3x640 -> @apply_op_config
+        , @match_conv_2d_nhwc_hwcf_2x128x128x320x3x3x320 -> @apply_op_config
+
+        // Batch matmul.
+        , @match_batch_matmul_64x968x320x640 -> @apply_op_config
+        , @match_batch_matmul_64x968x640x640 -> @apply_op_config
+        , @match_batch_matmul_64x968x320x960 -> @apply_op_config
+        , @match_batch_matmul_64x242x1280x1280 -> @apply_op_config
+        , @match_batch_matmul_64x242x640x960 -> @apply_op_config
+        , @match_batch_matmul_64x242x640x1280 -> @apply_op_config
+        , @match_batch_matmul_64x242x640x1920 -> @apply_op_config
+
+        // Contration.
```

Comment:
Who maintains this spec file? I see some typos in here and a good portion of this file looks generated?

Reply:
Looks like it mostly deals with the sdpa op and tunings. Looks like a few people have been committing to this file here: https://github.com/nod-ai/sdxl-scripts/blob/main/specs/attention_and_matmul_spec.mlir.